Makefile

```diff
@@ -285,7 +301,10 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 ##@ Templates
 
 .PHONY: cluster-e2e-templates
-cluster-e2e-templates: $(KUSTOMIZE) cluster-e2e-templates-v1beta1 cluster-e2e-templates-v1alpha4 ## Generate cluster templates for all versions
+cluster-e2e-templates: $(KUSTOMIZE) cluster-e2e-templates-v1beta1 cluster-e2e-templates-v1alpha4 cluster-e2e-templates-v124 ## Generate cluster templates for all versions
+
+cluster-e2e-templates-v124: $(KUSTOMIZE) ## Generate cluster templates for CAPX v1.2.4
```

Comment:
@jimmidyson This template used in E2EConfig. Maybe you right and it's better to download template from release, but I also prefer to make these changes in separate PR.

Reply:
As long as this doesn't get missed, I'm happy with a follow up PR. My reason for wanting to use release artifacts is so these old versions don't get mutated in this repo and skew e2e test results.

Reply:
But that means introducing these files in this PR, and then removing them in a follow up - it feels better to me to make this change in this PR rather than a follow up, but I won't block on that.

Reply:
Fixed

Reply:
@jimmidyson Sorry, some misunderstanding... The cluster-template.yaml manifest in release and manifest for E2E tests is not the same. Released manifest also requires patches. Find the way how to improve it is the case for future.

Reply:
So the released artifacts aren't usable directly? OK that is the context I'm missing! OK let's try to fix that up in future releases.